library/psa_crypto.c

```diff
@@ -1206,7 +1194,10 @@ psa_status_t psa_get_key_attributes( mbedtls_svc_key_id_t key,
                                 MBEDTLS_PSA_KA_MASK_DUAL_USE );
 
 #if defined(MBEDTLS_PSA_CRYPTO_SE_C)
-    if( psa_key_slot_is_external( slot ) )
+    psa_se_drv_table_entry_t *driver;
+
+    driver = psa_get_se_driver_entry( slot->attr.lifetime );
+    if( driver != NULL )
```

Comment:
If `driver` is only used here, can it not be inlined in the conditional statement?